examples/mnist_mlp_thresholdactiv.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+np.random.seed(1337) # for reproducibility
+
+from keras.datasets import mnist
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation
+from keras.layers.advanced_activations import TReLu, TLinear
+from keras.optimizers import SGD, Adam, RMSprop
+from keras.utils import np_utils
+
+'''
+    Train a simple deep NN on the MNIST dataset.
+
+    Get to 98.30% test accuracy after 20 epochs (there is *a lot* of margin for parameter tuning).
```

Comment:
The purpose was really just to provide an example of how to use it and verify operation -- indeed it does not perform better at this task.  I can remove the example if you prefer.